src/base.jl

```diff
@@ -28,23 +28,17 @@ end
 
 Checks if all matrices of a Kronecker product are square.
 """
-function issquare(K::AbstractKroneckerProduct)
-    return issquare(K.A) && issquare(K.B)
-end
+issquare(K::AbstractKroneckerProduct) = issquare(K.A) && issquare(K.B)
 
-function squarecheck(K::AbstractKroneckerProduct)
-    issquare(K) || throw(DimensionMismatch("kronecker system is not composed of two square matrices: $size(K.A) and $size(K.B)"))
-end
+squarecheck(K::AbstractKroneckerProduct) = issquare(K) || throw(DimensionMismatch(
+           "kronecker system is not composed of two square matrices: $size(K.A) and $size(K.B)"))
```

Comment:
Sorry to be picky, but this line still overruns the 92 char limit. Could it please be de-indented to 4 chars, rather than 11?